uui-components/src/inputs/DatePicker/helpers.ts

```diff
@@ -32,5 +41,15 @@ export const toValueDateFormat = (value: string, format?: string): string => {
 export const toCustomDateFormat = (value: string, format?: string): string => {
     const customFormat = format || defaultFormat;
 
-    return value ? dayjs(value, supportedDateFormats(format), true).format(customFormat) : null;
+    if (value) {
+        const dayjsObj = dayjs(value, supportedDateFormats(format), true);
+
+        if (dayjsObj.isValid()) {
+            return dayjsObj.format(customFormat);
+        } else {
+            return null;
+        }
+    } else {
+        return null;
+    }
```

Comment:
This function returned `Invalid date` previously in case of value is a partial input in date picker. Now it returns `null`. Is that narmal breaking change @AlekseyManetov ?

Reply:
Done, helpers are not exported